api/android/api/src/main/jni/nnstreamer-native-api.c

```diff
@@ -635,6 +635,118 @@ nns_get_nnfw_type (jint fw_type, ml_nnfw_type_e * nnfw)
   return is_supported && ml_nnfw_is_available (*nnfw, ML_NNFW_HW_ANY);
 }
 
+/**
+ * @brief Set additional environment
+ */
+static gboolean
+nns_set_env (JNIEnv * env, jobject context)
```

Comment:
@jaeyun-jung / @anyj0527 : SNPE-context in Native-API.c (common framework code) is NOT a good practice. This code must be moved to ext/fitler/tensor-filter/...SNPE... 

Please confirm this and then, merge this PR. (and create another PR later fixing this architectural error.)